frontend/express/public/javascripts/countly/vue/core.js

```diff
@@ -121,6 +121,27 @@
                         }
                     }
                 }
+            },
+            countlySidebar: {
```

Comment:
That's totally fine, I mean registering the sidebar in core, as it is a permanent thing, that doesn't need to be unregistered anytime.